content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md

```diff
@@ -54,6 +54,15 @@ the corresponding `PersistentVolume` is not be deleted. Instead, it is moved to
 
     where `<your-pv-name>` is the name of your chosen PersistentVolume.
 
+    {{< note >}}
+    If you are running `kubectl` on Windows, you'd need to use double quotes on the
+    command line and thus escape the inner double quotes like this:
+
+    ```shell
+    kubectl patch pv <your-pv-name> -p "{\"spec\":{\"persistentVolumeReclaimPolicy\":\"Retain\"}}"
```

Comment:
Maybe remove the three-backtick lines before and after, and change this to:
`kubectl patch pv <your-pv-name> -p "{\"spec\":{\"persistentVolumeReclaimPolicy\":\"Retain\"}}"`

Reply:
Thanks @sftim , I actually noticed the page broke after the last change, so I started testing locally with the kubernetes-hugo docker image. I'll see how it looks without the backticks and if the page remains in tact and will push the latest change afterwards, thanks!

Reply:
@sftim it renders correctly now with the latest commit